renderers/video_renderer_rpi.c

```diff
@@ -117,7 +133,7 @@ int video_renderer_init_decoder(video_renderer_t *renderer) {
 
     bcm_host_init();
 
-    if (renderer->background) video_renderer_draw_background();
```

Comment:
For the case that the user chooses to always draw the background, please move the call to video_renderer_render_background into rpiplay.cpp, to the place immediately after the video renderer is initialised. That way, it's always rpiplay.cpp's responsibility to manage the background, instead of having it spread around in rpiplay.cpp and the renderer itself. This should also allow us to remove the background argument to the renderer initialisation.

Reply:
It's been done in latest commit

Reply:
In latest commit, there is a new function `video_renderer_update_background` will be called in video_renderer_init to support `always` background drawing mode. 

`rpiplay.cpp` is responsable to call `video_renderer_update_background` in `conn_init` and `conn_destroy` that seems only can be set to raop_t instance in `start_server` of `rpiplay.cpp`. 

If there is better way to move all background details to video renderer, please tell me, I will do it.